src/frontend/src/optimizer/rule/table_function_to_file_scan_rule.rs

```diff
@@ -106,6 +107,25 @@ impl Rule for TableFunctionToFileScanRule {
                     )
                     .into(),
                 )
+            } else if "azblob".eq_ignore_ascii_case(&eval_args[1]) {
+                let account_name = eval_args[2].clone();
+                let account_key = eval_args[3].clone();
+                let endpoint = eval_args[4].clone();
```

Comment:
To make it more aligned with `s3`, I think we should put endpoint before account_name and account_key, i.e. file_scan('parquet', 'azblob', endpoint, account_name, account_key, file_location).